macos/FluentUI/DatePicker/CalendarView.swift

```diff
@@ -172,6 +172,10 @@ struct CalendarDay {
 
 	/// String used for accessibility scenarios, like VoiceOver
 	let accessibilityLabel: String
+	
+	/// String with extra descriptors for accessibility scenarios.  VoiceOver
+	/// reads this string after a short delay if focus stays on an element.
+	let accessibilityHelp: String?
 
 	/// Secondary String representation of the day
 	let secondaryLabel: String?
```

Comment:
the secondaryLabel can't be just used for accessibilityHelp for date?

Reply:
The secondaryLabel seems to have a very different purpose.  I'm not entirely clear on the use case but I figured I'd better not mess with it.  Here's what it looks like when I enable the "secondaryCalendar" checkbox in the test app:

<img width="271" alt="Screen Shot 2021-04-15 at 8 47 16 PM" src="https://user-images.githubusercontent.com/67027949/114968753-d8063580-9e2b-11eb-81a2-dec784087c06.png">

Reply:
I suppose it must be for cross-referencing dates in different calendars.  In the test app example it's Chinese Lunar vs. Gregorian... you can tell that the numerals don't sync up with each other so it's not a simple relabeling.  Anyway `secondaryLabel` stores the actual strings that go on screen in that secondary position, so no we can't use it to store the weekdays.

Reply:
thanks for clarifying this. do you mind updating the comment for this variable for future reference?

Reply:
secondaryLabel was for people who wanted to put a second calendar, such as one with Chinese holidays on it (I think that was the initial ask). There might be some better documentation in the actual PR @lcapkovic had for initially checking in the DatePicker the first time around.

Reply:
> thanks for clarifying this. do you mind updating the comment for this variable for future reference?

Done